lib/bindings/python/src/dynamo/replay/main.py

```
@@ -106,6 +106,100 @@ def _load_aic_perf_config(args: argparse.Namespace):
     )
 
 
+def _engine_caps(args: MockEngineArgs) -> "EngineCapabilities":
```

Ruff (0.15.10)

[error] Undefined name `EngineCapabilities`

(F821)